File: S04/prob03.py
# -*- coding: utf-8 -*-
"""
Created on Tue Jul 19 16:03:37 2022

@author: Gerardo A. Rivera Tello
@handle: DangoMelon
"""

n = eval(input("Ingrese la cantidad de alumnos: "))

_n = 0
jalo = 0
aprobo = 0
while _n != n:
    nota = eval(input("Ingrese la nota del alumno: "))
    if nota >= 0 and nota <= 20:
        _n += 1
        if nota < 11:
            jalo += 1
        else:
            aprobo += 1
    else:
        print("La nota ingresada no es valida")

print("La cantidad de alumnos que no aprobaron es: ", jalo)
print("La cantidad de alumnos que aprobaron es: ", aprobo)

File: S04/prob03.cpp
/*
 * Created on Tue Jul 19 16:07:04 2022

 * @author: Gerardo A. Rivera Tello
 * @handle: DangoMelon
 */

#include <iostream>

int main()
{
    int alumnos, aprobado = 0, desaprobado = 0;
    int n = 0;

    float nota;

    std::cout << "Ingrese la cantidad de alumnos: ";
    std::cin >> alumnos;

    while (n != alumnos)
    {
        std::cout << "Ingrese la nota del alumno: ";
        std::cin >> nota;

        if (nota >= 0 && nota <= 20)
        {
            n++;
            if (nota > 11)
            {
                aprobado++;
            }
            else
            {
                desaprobado++;
            }
        }
        else
            std::cout << "La nota ingresada no es valida" << std::endl;
    }

    std::cout << "La cantidad de alumnos aprobados es: " << aprobado << std::endl;
    std::cout << "La cantidad de alumnos desaprobados es: " << desaprobado << std::endl;
}